docs/static_site/src/pages/api/julia/index.md

```diff
@@ -35,5 +35,6 @@ computing and the state-of-art deep learning to Julia.
   and apply them to tasks such as image classification and data science challenges.
 
 ## Installation
-* [Ubuntu installation guide]({{'/get_started/ubuntu_setup' | relative_url}})
-* Mac / Windows guides are not available (contributions welcome!)
+* [Ubuntu installation guide]({{'/get_started/ubuntu_setup.html'|relative_url}})
```

Comment:
Same here (+ several more on other files, wherever `.html` is added without changing the base route).